pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -494,16 +494,16 @@ func TestPreFilterPlugins(t *testing.T) {
 		f.RunPreFilterExtensionAddPod(nil, nil, nil, nil)
 		f.RunPreFilterExtensionRemovePod(nil, nil, nil, nil)
 
-		if preFilter1.PreFilterCalled != 1 {
+		if preFilter1.PreFilterCalled != 0 {
```

Comment:
why making this change?

Reply:
This is related with https://github.com/kubernetes/kubernetes/pull/83538#discussion_r332316045